bottomless/src/backup.rs

```diff
@@ -49,15 +47,9 @@ impl WalCopier {
             tracing::trace!("Trying to flush empty frame range");
             return Ok(frames.start - 1);
         }
-        let wal = {
-            if self.wal.is_none() {
```

Comment:
This part was the issue I guess?

Reply:
yep, that's it

Reply:
I kept the commit history separate, so you can skip the first one since it's just reorganizing everything to more closely match the SQLite docs description, commits 2 & 3 are key